k8s/supportability/src/collect/logs/mod.rs

```diff
@@ -109,95 +113,114 @@ impl LogCollection {
         }))
     }
 
-    async fn get_logging_resources(
+    async fn pod_logging_resources(
         &self,
-        pods: Vec<Pod>,
+        pod: Pod,
+        nodes_map: HashMap<String, Node>,
```

Comment:
We can pass by reference?

```suggestion
        nodes_map: &HashMap<String, Node>,
```